examples/serverless-websocket/README.md

```diff
@@ -0,0 +1 @@
+This is an example usage of laconia with serverless framework.
```

Comment:
Can you add a couple of lines here explaining what can users find in this example:

Reply:
done, hope this is sufficient now.